runner/views/pipeline_view.py

```diff
@@ -16,9 +17,16 @@ class PipelineViewSet(mixins.ListModelMixin,
                       mixins.CreateModelMixin,
                       mixins.RetrieveModelMixin,
                       GenericViewSet):
-    queryset = Pipeline.objects.order_by('id').all()
+    queryset = Pipeline.objects.all()
```

Comment:
Should we keep some kind of ordering here?

Reply:
It's your call, I didn't see a point, but without it objects are not guaranteed to come from the DB in the same order on each request.

Reply:
You are right. Ordering for pipelines doesn't actually make a lot of sense. If we need it in the future we can easily add it.